pandas/tests/window/test_timeseries_window.py

```diff
@@ -721,7 +721,7 @@ def test_rolling_on_decreasing_index(self):
         df = DataFrame({"column": [3, 4, 4, 2, 1]}, index=reversed(index))
         result = df.rolling("2s").min()
         expected = DataFrame(
-            {"column": [3.0, 3.0, 3.0, 2.0, 1.0]}, index=reversed(index)
+            {"column": [3.0, 4.0, 4.0, 2.0, 1.0]}, index=reversed(index)
```

Comment:
The original expected value was wrong: https://github.com/pandas-dev/pandas/pull/28297#discussion_r351669004.